addons/console/Console.gd

```diff
@@ -164,18 +166,18 @@ func toggle_size() -> void:
 	else:
 		control.anchor_bottom = 1.0
 
-
 func toggle_console() -> void:
 	control.visible = !control.visible
 	if (control.visible):
-		get_tree().paused = true
+		was_paused_already = get_tree().paused
+		get_tree().paused = was_paused_already or pause_enabled
 		line_edit.grab_focus()
 		emit_signal("console_opened")
 	else:
 		control.anchor_bottom = 1.0
 		scroll_to_bottom()
 		reset_autocomplete()
-		get_tree().paused = false
+		get_tree().paused = was_paused_already
```

Comment:
Could maybe do
if (pause_enabled && !was_paused_already):
    get_tree().paused = false

Reply:
Seems to be working for most common sense behaviours.

Got problems with the following:
- Open console (game pauses)
- Somehow the pause menu opens at a later time (doesn't make sense in a usual game because it is paused)
- Close console (game unpauses but pause menu is still visible)

But that's a fabricated very unlikely case imho.